docs/fundamentals/codespaces.md

```diff
@@ -0,0 +1,130 @@
+---
```

Comment:
Does codespaces belong under "fundamentals"? That doesn't seem like an appropriate place for it.

![image](https://github.com/user-attachments/assets/61ff344f-6ba8-43bc-9808-e743a7f8541b)

The other things are fundamental to Aspire.

Maybe these docs could be under "Get Started" => "Setup and tooling"?